### DISCUSSION TITLE
Autorec

### DISCUSSION THREAD

**Discussion by Goulougba** [helgoboss/helgobox#1154] [Q&A]
Hi, for a sound installation I would like to automatically record when someone is singing in a mic. So I put a ReaGate on a track, which I affect to a midi canal, and I tried to control the record button with it. But I don't know why it did not work as I thought. ReaLearn detect my voice and the gate works, but I can't control the record button the right way. I tried severals options : record/stop, play/pause etc. but I couldn't find the right settings and I could not find any answer in this forum. I am sorry if I missed something somewhere.
These are my settings. Does anyone have an idea to help ? thanks a lot.
<img width="1280" alt="Capture d’écran 2024-08-30 à 23 48 25" src="https://github.com/user-attachments/assets/2e5042b4-0eb1-4d20-8f40-51f28ce8be51">

**Comment by helgoboss** [comment]
I think it's because ReaGate fires 2 messages, one NOTE ON when above threshold and one NOTE OFF when below threshold? So you are starting recording and in the next moment stop it again.

You might want to set "For buttons => Press & release" to "Press only".
